core/backend/src/IModelHost.ts

```diff
@@ -404,23 +404,21 @@ export class IModelHost {
     if (options.crashReportingConfig && options.crashReportingConfig.crashDir && !ProcessDetector.isElectronAppBackend && !ProcessDetector.isMobileAppBackend) {
       this.platform.setCrashReporting(options.crashReportingConfig);
 
-      Logger.logTrace(loggerCategory, "Configured crash reporting", {
+      Logger.logTrace(loggerCategory, "Configured iModel crash reporting", {
```

Comment:
I'd leave the message as it was previously since the logger category will give the context needed.

Reply:
Done